docs/reference/aggregations/bucket/geohashgrid-aggregation.asciidoc

```diff
@@ -15,7 +15,9 @@ Geohashes used in this aggregation can have a choice of precision between 1 and
 WARNING: The highest-precision geohash of length 12 produces cells that cover less than a square metre of land and so high-precision requests can be very costly in terms of RAM and result sizes.
 Please see the example below on how to first filter the aggregation to a smaller geographic area before requesting high-levels of detail.
 
-The specified field must be of type `geo_point` (which can only be set explicitly in the mappings) and it can also hold an array of `geo_point` fields, in which case all points will be taken into account during aggregation.
+The specified field must be of type `geo_point` or `geo_shape` (which can only be
+set explicitly in the mappings). It can also hold an array of `geo_point` fields,
+in which case all points will be taken into account during aggregation.
```

Comment:
I wonder if this paragraph is needed. We already state that the agg works on `geo_point` and `geo_shape` fields. The note about explicit mappings seems more relevant to the field types than the agg.

Do users commonly use `geo_point` to store arrays? If so, would they assume array values are supported (as they are elsewhere in ES)?

I included a suggestion if we decide to keep the para. Feel free to ignore if wanted.
```suggestion
You can only use `geotile_grid` to aggregate an explicitly mapped `geo_point` or
`geo_shape` field. If the `geo_point` field contains an array, `geotile_grid`
aggregates all the array values.
```